### PR TITLE
[DOC] Update & Simplify informations

### PR DESCRIPTION
**Pull request by villander**
This PR update information from 2019 to 2020, simplifies README. Also, add changelog missing to `v0.8.5`

### CODE REVIEW

**Comment by villander** [comment]
cc: @dgeb @rwjblue

File: README.md [hunk 1]
```diff
@@ -5,79 +5,40 @@ RFC](https://github.com/emberjs/rfcs/blob/master/text/0010-engines.md). Engines
...
-- [Testing](http://ember-engines.com/guide/testing)
-
-**Other Resources**
-
-- [Engine API Documentation](https://api.emberjs.com/ember/release/classes/Engine)
-- [Engines 1.0 Roadmap](https://discuss.emberjs.com/t/engines-1-0-roadmap/14914)
-
-## Introduction Video
-
-[![Introduction to Ember Engines at Global Ember Meetup](https://i.vimeocdn.com/video/559400541_640x360.jpg)](https://vimeo.com/157688181)
```
Comment (rwjblue) [inline README.md:39]: Why delete these links?

Comment (villander) [reply README.md:39]: Because we're linking the Official guides. Like https://github.com/emberjs/ember.js already did.

**Comment by ghaagsma** [comment]
Is there anything blocking this change from being reviewed/merged? The links to dead pages in the README don't inspire confidence in the library, and it seems like relatively low-hanging fruit to fix at this point.

**Pull request merged**